packages/sanity/src/desk/i18n/resources.ts

```diff
@@ -116,6 +116,22 @@ const deskLocaleStrings = {
   /** Message prompting the user to confirm that they want to restore to an earlier version*/
   'action.restore.confirm-dialog.confirm-discard-changes':
     'Are you sure you want to restore this document?',
+
+  /** --- REVIEW CHANGES BUTTON --- */
+  /** Label for button when status is syncing */
+  'status-bar.review-changes-button.status.syncing.text': 'Saving...',
+
+  /** Label for button when status is saved */
+  'status-bar.review-changes-button.status.saved.text': 'Saved!',
+
+  /** Primary text for tooltip for the button */
+  'status-bar.review-changes-button.tooltip.text': 'Review changes',
+
+  /** Text for the secondary text for tooltip for the button */
+  'status-bar.review-changes-button.tooltip.changes-saved': 'Changes saved',
```

Comment:
For consistency I guess the `.tooltip` should be at the end?

Reply:
maybe? it's the same tool tip, but this tool tip has a primary and secondary text.